src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellFlyoutRenderer.cs

```diff
@@ -124,7 +124,7 @@ void IFlyoutBehaviorObserver.OnFlyoutBehaviorChanged(FlyoutBehavior behavior)
 		int _flyoutWidthDefault;
 		double _flyoutWidth = -1;
 		double _flyoutHeight;
-		bool _flyoutFirstDrawPassFinished;
+		internal bool FlyoutFirstDrawPassFinished { get; private set; }
```

Comment:
Why was this change made?

Reply:
This is used by the unit tests to validate a certain point in the life cycle

https://github.com/dotnet/maui/pull/8022/files/43b2c4d6f6d70ff46432548abba53ebe8526224d#diff-682cc392a19d37f2fba78a5128ff76b224290d624f02e9e1020eaa2a882ad9d1R249

Reply:
just trying to understand the change to `internal` and `private set` better - why would the original bool not have sufficed?